classes/container.js

```diff
@@ -0,0 +1,48 @@
+class Container
```

Comment:
TODO: A way to remove products from a container.

Have this be done automatically when the quantity on any product drops to zero (could be done when calculating the total used capacity).